Lib/dataclasses.py

```diff
@@ -188,7 +188,8 @@ class _MISSING_TYPE:
 # A sentinel object to indicate that following fields are keyword-only by
 # default.  Use a class to give it a better repr.
 class _KW_ONLY_TYPE:
-    pass
+    def __call__(self, *args, **kwds):
```

Comment:
Is this necessary? Looks like something that get_type_hints() would need.

Reply:
Yes, this is needed because the code path that checks `callable()` is run as long as the typeform is contained within `Annotated`. So without this, you get runtime errors.

This particular one is only needed if we want to support `Annotated[KW_ONLY, ...]`. Let's discuss that below.